### PR TITLE
Updated OnlineAudioFile PS cmdlet description

### PR DESCRIPTION
**Pull request by JensTrier**
Updated file for Import-CsOnlineAudioFile and added Get/Remove-CsOnlineAudioFile PS cmdlet descriptions

### CODE REVIEW

File: skype/skype-ps/skype/Remove-CsOnlineAudioFile.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+Marks an audio file of application type TenantGlobal for deletion and later removal (within 24 hours).
+
+> [!NOTE]
+> **Preview** The use of this cmdlet is in Public Preview.
+
+
+## SYNTAX
+
+```powershell
+Remove-CsOnlineAudioFile -Identity] <string>  [<CommonParameters>]
```
Comment (get-itips) [inline skype/skype-ps/skype/Remove-CsOnlineAudioFile.md:24]:
```suggestion
Remove-CsOnlineAudioFile -Identity <string>  [<CommonParameters>]

```

File: skype/skype-ps/skype/Remove-CsOnlineAudioFile.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+## INPUTS
+
+### None
+
+## OUTPUTS
+
+### System.Object
+
+## NOTES
+Please note that using this cmdlet on other application types like OrgAutoAttendant and HuntGroup does not make the audio file for deletion. These kind of audio files will automatically be deleted, when
```
Comment (get-itips) [inline skype/skype-ps/skype/Remove-CsOnlineAudioFile.md:64]:
```suggestion
Please note that using this cmdlet on other application types like OrgAutoAttendant and HuntGroup does not mark the audio file for deletion. These kinds of audio files will automatically be deleted, when

```

**Review by get-itips — APPROVED** [review]
Some minor edits

File: skype/skype-ps/skype/Get-CsOnlineAudioFile.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+Required: False
+Position: Named
+Default value: TenantGlobal
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Identity
+The Id of the specific audio file that you would like to see information about. If you are only specifying -Identity the -ApplicationId it is assumed to be TenantGlobal.
```
Comment (dariomws) [inline skype/skype-ps/skype/Get-CsOnlineAudioFile.md:91]:
```suggestion
The Id of the specific audio file that you would like to see information about. If you are only specifying -Identity, the -ApplicationId it is assumed to be TenantGlobal. 

```

File: skype/skype-ps/skype/Get-CsOnlineAudioFile.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+Required: False
+Position: Named
+Default value: TenantGlobal
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Identity
+The Id of the specific audio file that you would like to see information about. If you are only specifying -Identity the -ApplicationId it is assumed to be TenantGlobal. 
+If you need to see information of a specific audio file with ApplicationId of OrgAutoAttendant or HuntGroup you need to specify -ApplicationId with the corresponding value and -Identity with the Id of the audio file.
```
Comment (dariomws) [inline skype/skype-ps/skype/Get-CsOnlineAudioFile.md:92]:
```suggestion
If you need to see the information of a specific audio file with ApplicationId of OrgAutoAttendant or HuntGroup, you need to specify -ApplicationId with the corresponding value and -Identity with the Id of the audio file.

```

File: skype/skype-ps/skype/Get-CsOnlineAudioFile.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+
+### Example 1
+```powershell
+Get-CsOnlineAudioFile
+
+Id                : 85364afb59a143fc9466979e0f34f749
+FileName          : CustomMoH.mp3
+ApplicationId     : TenantGlobal
+MarkedForDeletion : False
+```
```
Comment (dariomws) [inline skype/skype-ps/skype/Get-CsOnlineAudioFile.md:39]: For examples with output, please use the following format:
https://docs.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide#executable-examples

**Review by dariomws — APPROVED** [review]
Minor suggestions and I approve because it's on preview but I found the following error in 2.5.1 and 2.4.1-preview:
image

File: skype/skype-ps/skype/Get-CsOnlineAudioFile.md (hunk 3 above)
Comment (JensTrier) [reply skype/skype-ps/skype/Get-CsOnlineAudioFile.md:91]: @dariomws thanks! I can repro the error in another test tenant, will investigate

**Comment by yogkumgit** [comment]
@JensTrier

**Comment by JensTrier** [comment]
> Minor suggestions and I approve because it's on preview but I found the following error in 2.5.1 and 2.4.1-preview:
> image

The error is caused by having no audio files uploaded. We are fixing the output before GA

**Comment by JensTrier** [comment]
> @JensTrier

Did I do what you wanted me to do?

**Comment by dariomws** [comment]
@JensTrier thank!

Can I help you? What are you trying to do?

**Comment by JensTrier** [comment]
> @JensTrier thanks!
> 
> Can I help you? What are you trying to do?

Nothing really :-) I just thought you wanted me to do something when you at mentioned above. I took it as you wanted me to commit the suggestion changes and additions so I did that.

**Comment by yogkumgit** [comment]
@tseward

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**